# HG changeset patch
# User Robert Miller <rlmillster@gmail.com>
# Date 1171818771 28800
# Node ID dbb606b4646cbc744b1c8f2db94a637c35cec6a9
# Parent  5e710fb1ae45d58847e761b96ab8dc33a6c9f483
Tachyon now shows outside of the notebook

diff -r 5e710fb1ae45 -r dbb606b4646c sage/plot/tachyon.py
--- a/sage/plot/tachyon.py	Sun Feb 18 09:06:42 2007 -0800
+++ b/sage/plot/tachyon.py	Sun Feb 18 09:12:51 2007 -0800
@@ -222,7 +222,9 @@ class Tachyon(SageObject):
             filename = 'sage%s.png'%i
             self.save(filename, verbose=verbose, extra_opts=extra_opts)
         else:
-            raise NotImplementedError
+            filename = sage.misc.misc.tmp_filename() + '.png'
+            self.save(filename, verbose=verbose, extra_opts=extra_opts)
+            os.system('%s %s 2>/dev/null 1>/dev/null &'%(sage.misc.viewer.browser(), filename))
 
 
     def _res(self):
